config/locales/en/dit_landing_page.yml

```diff
@@ -3,191 +3,36 @@ en:
-
-      UK legal professionals should contact their local regulators to understand the rules on third country lawyers in each member state of the EU. [Find out more about UK lawyers practising in the EU from 1 January 2021.](/government/publications/uk-lawyers-practising-in-the-eu-norway-iceland-or-liechtenstein-from-1-january-2021/uk-lawyers-practising-in-the-eu-norway-iceland-or-liechtenstein-after-1-january-2021)
-
-      ### Business activity in the UK
-
-      From 1 January 2021, freedom of movement between the EU and UK will end.
-
-      EU, EEA and Swiss citizens entering the UK for work purposes may need to apply for a visa through the UK’s points-based immigration system. This depends on the nature of their visit.
-
-      For visits under 6 months, EU, EEA and Swiss citizens will be able to enter the UK without applying for a visa. They may participate in business-related activities, such as meetings, events and conferences.
-
-      [Find out more about permitted business-related activity for visits to the UK under 6 months.](/guidance/immigration-rules/immigration-rules-appendix-v-visitor-rules)
-
-      If you require EU, EEA or Swiss citizens to go to the UK to work for longer than 6 months, you’ll need to check the UK’s immigration laws.
-
-      [Find out more about the UK’s immigration laws.](/guidance/new-immigration-system-what-you-need-to-know)
-
-      If you employ or intend to employ an EU, EEA or Swiss citizen to commute into the UK, you’ll need to consult guidance for frontier workers.
-
-      [Find out more about frontier working into the UK.](/guidance/frontier-workers-in-the-uk-rights-and-status)
-
-      The new rules do not apply to Irish citizens because of the Common Travel Area arrangement.
-
-      ### Copyright and unregistered designs
-
-      Changes to EU cross-border copyright mechanisms may impact your business. Make sure you have the right permissions from 1 January 2021.
-
-      [Check changes to copyright law from 1 January 2021.](/guidance/changes-to-copyright-law-after-the-transition-period)
-
-      You may not have protection for unregistered designs if you do not disclose your designs correctly.
-
-      [Check the new rules for unregistered design protection in the UK and EU from 1 January 2021.](/guidance/changes-to-unregistered-designs-after-the-transition-period)
-
-    training_section_title: Additional help and support
+      If you run an EU-based business, you need to check the new rules and prepare for the changes so that you can continue trading with the UK from January 2021.
+    guidance_links:
+      - label: Importing from the UK
+        url: "/guidance/eu-business-importing-from-the-uk"
+        text: Find out what you need to do to import from the UK, including customs procedures and importing goods such as food, drink, animal, and plant products.
+      - label: Exporting to the UK
+        url: "/guidance/eu-business-exporting-to-the-uk"
```

Comment:
Each of the `guidance_links` is for a page that has a translation nav itself. Not a blocker for now, but it would be nice if when a user on https://www.gov.uk/eubusiness.de clicks on this link, they are taken to https://www.gov.uk//guidance/eu-business-exporting-to-the-uk.de instead of the english language version. I haven't tried it locally but I reckon we could do that quite easily by appending the locale to the urls.